contracts/Migrations.sol

```diff
@@ -3,24 +3,24 @@ pragma solidity ^0.4.17;
 
 contract Migrations {
 	address public owner;
-	uint public last_completed_migration;
+	uint public lastCompletedMigration;
```

Comment:
This probably breaks truffle's migration system since it expects a getter with this name.

Reply:
I think maybe we should just disable linting for Migrations.sol altogether...